config/src/lib.rs

```diff
@@ -180,7 +180,7 @@ pub struct RelayerConfig {
     /// Bootstrap servers that the peer should connect to
     pub bootstrap_servers: Vec<(WrappedPeerId, Multiaddr)>,
     /// The cluster keypair
-    pub cluster_keypair: Keypair,
+    pub cluster_keypair: DalekKeypair,
```

Comment:
nit: can we make sure there's a `TODO` somewhere (even internally) to switch to Bn254 ECDSA keys for the cluster?

Reply:
part of a broader TODO with fees, this keypair is more for gossip